packages/@okta/vuepress-site/docs/reference/api/events/index.md

```diff
@@ -30,25 +30,25 @@ Fetches a list of events from your Okta organization system log
 | :---------- | :-------------------------------------------------------------------------------------------- | :----------- | :--------- | :--------- | :------ |
 | limit       | Specifies the number of results to page                                                       | Query        | Number     | FALSE      | 1000    |
 | startDate   | Specifies the timestamp to list events after                                                  | Query        | Date       | FALSE      |         |
-| filter      | [Filter expression](/docs/reference/api-overview/#filter) for events         | Query        | String     | FALSE      |         |
+| filter      | [Filter expression](/docs/reference/core-okta-api/#filter) for events         | Query        | String     | FALSE      |         |
 | after       | Specifies the pagination cursor for the next page of events                                   | Query        | String     | FALSE      |         |
 
 Parameter Details
 
-* Treat the `after` cursor as an opaque value as its contents are subject to change without notice. Obtain it through the `next` link relation. See [Pagination](/docs/reference/api-overview/#pagination) for more details on link relations.
+* Treat the `after` cursor as an opaque value as its contents are subject to change without notice. Obtain it through the `next` link relation. See [Pagination](/docs/reference/core-okta-api/#pagination) for more details on link relations.
 * `startDate` and `filter` query parameters are mutually exclusive and cannot be used together in the same request.
 * `startDate` and `after` query parameters are mutually exclusive and cannot be used together in the same request.
 * `startDate` defaults to 1 hour ago when `filter`, `after` and `startDate` query parameters are omitted.
 * `limit` can be no larger than 1000
 
 ###### Reliable Ingestion
 
-The most reliable method to ingest all events from Okta is to use a [pagination](/docs/reference/api-overview/#pagination) cursor via the `after` parameter. This will ensure that events are not skipped or duplicated due to the lack of timestamp precision.
+The most reliable method to ingest all events from Okta is to use a [pagination](/docs/reference/core-okta-api/#pagination) cursor via the `after` parameter. This will ensure that events are not skipped or duplicated due to the lack of timestamp precision.
```

Comment:
```suggestion
The most reliable method to ingest all events from Okta is to use a [pagination](/docs/reference/core-okta-api/#pagination) cursor through the `after` parameter. This will ensure that events are not skipped or duplicated due to the lack of timestamp precision.
```